activesupport/lib/active_support/local_store.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module ActiveSupport
+  module LocalStore # :nodoc:
+    module FiberLocal
+      def current
+        Thread.current[:active_support_execution_context] ||= {}
```

Comment:
Why are the keys mentioning the execution context?

```suggestion
        Thread.current[:active_support_local_store] ||= {}
```

Reply:
Good catch ❤️ , I forgot to properly rename them when I split this PR in two. I'll fix it.